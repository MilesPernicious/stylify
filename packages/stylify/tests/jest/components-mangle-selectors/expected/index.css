.s0,
.s8{
	padding:8px
}
.s1,
.s8{
	background-color:#000
}
.s2,
.s8{
	display:inline-block
}
.s3,
.s8,
.s9{
	font-size:24px
}
.s4,
.s10{
	max-width:800px
}
.s5,
.s10{
	margin:0 auto
}
.s6,
.s9{
	color:green
}

@media (min-width: 768px) {
.s7,
.s9{
	font-size:32px
}
}