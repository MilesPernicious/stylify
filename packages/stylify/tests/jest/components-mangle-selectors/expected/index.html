<button class="s8"></button>
<div class="s10"></div>
<h1 class="s9"></h1>
